Playbooks/playbook-Prisma_Cloud_Remediation_-_AWS_Inactive_users_for_more_than_30_days.yml

```diff
@@ -0,0 +1,526 @@
+id: Prisma Cloud Remediation - AWS Inactive users for more than 30 days
+version: -1
+name: Prisma Cloud Remediation - AWS Inactive users for more than 30 days
```

Comment:
Playbook name - every word should start with a capital letter.
Make sure to change the `id` field accordingly.